src/eradiate/_mode.py

```diff
@@ -14,52 +14,51 @@
 # ------------------------------------------------------------------------------
 
 
-class ModeFlags(enum.Flag):
+class Flag(enum.Flag):
     """
-    Flags defining Eradiate mode features.
+    Small extension to :class:`enum.Flag` that adds a ``convert()`` class method
+    constructor.
     """
 
-    # -- Mode feature flags ----------------------------------------------------
```

Comment:
I like how the new flag classes removes the bitwise logic, its easier to read. It looks easier to maintain too

Reply:
Yes, mode definitions are clearer and more streamlined. This refactoring was long overdue!